guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -47,6 +47,12 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
     config.controller = ['$scope', '$injector', '$element',
             function guacClipboardController($scope, $injector, $element) {
 
+        /**
+         * Set to true if the clipboard data should be displayed in the  
```

Comment:
You might consider rewording this comment to be about what this variable _does_ not what the code should set it to.